.env.dist

```diff
@@ -38,3 +38,7 @@ DATABASE_URL="mysql://db_user:db_password@127.0.0.1:3306/db_name?serverVersion=5
 # MESSENGER_TRANSPORT_DSN=amqp://guest:guest@localhost:5672/%2f/messages
 # MESSENGER_TRANSPORT_DSN=redis://localhost:6379/messages
 ###< symfony/messenger ###
+
+###> symfony/mailer ###
+MAILER_DSN=niks
```

Comment:
removen, staat al in .env

Reply:
Moet ik deze niet laten staan voor andere mensen. Zodat ze deze over kunnen nemen als ik mijn code push?. (als deze niet aanstaat kan je niet de ww vergeten pagina bezoeken)

Reply:
Line 23 t/m 25:
###> symfony/mailer ###
# MAILER_DSN=smtp://localhost
###< symfony/mailer ###

Kan je beter deze uncommenten ipv nogmaals toevoegen